frontend/utils/DateUtils.js

```diff
@@ -0,0 +1,53 @@
+import moment from 'moment';
+
+function categorizeDate(creationDate, currentDate) {
```

Comment:
Let's stick to a single pattern on how we are creating functions, in this case we use `const categorizeData = () => {}`